### DISCUSSION TITLE
http://localhost:8080/

### DISCUSSION THREAD

**Discussion by giscus** [y9san9/public-vault#6] [Announcements]
# http://localhost:8080/

Hi, my name is Aλex Follow through this website to learn something about me. Bio I don’t know how to summarize me Full Name: Alexander Sokolinskiy (Александр Соколинский) Preferred Name: Alex Sokol Unique Name: y9san9 (see pronunciation and origin) Timezone: UTC+3 Birth date: 20.12.2003 Feel free to reach me: Telegram: @y9san9 Email: y9san9@gmail.com GitHub: @y9san9 In any of these languages: 🇺🇸 English (Fluent) 🇷🇺 Russian (Native) 🇺🇦 Ukrainian (Limited) Shared Mutable State Hard Skills Personal Projects Public Activity Education Political Views .

http://localhost:8080/

**Comment by y9san9** [comment]
Hello!
